scalableminds/webknossos
# Use zip name when uploading hybrid/volume/collection

### URL of deployed dev instance (used for testing):
- https://zipuploadusename.webknossos.xyz

### Steps to test:
- upload zipped annotation, zip name should be used for the annotation
- upload single nml (no zip), its name should still be used for the annotation

### Issues:
- fixes #4170 

------
- [ ] waiting for lab feedback https://discuss.webknossos.org/t/use-name-of-zip-file-as-name-for-hybrid-tracing/1511
